webapp/src/components/backstage/runs_list/runs_list.tsx

```diff
@@ -19,7 +19,7 @@ interface Props {
     totalCount: number
     fetchParams: FetchPlaybookRunsParams
     setFetchParams: React.Dispatch<React.SetStateAction<FetchPlaybookRunsParams>>
-    filterPill: React.ReactNode
+    filterPill: React.ReactNode | null
```

Comment:
Unnecessary since ReactNode already includes null.
```suggestion
    filterPill: React.ReactNode
```

Reply:
Technically yes, and I would much prefer that, but seems react-devtools-backend isn't quite smart enough to infer that. It converts:
- `{filterPill: React.ReactNode}` to be `{filterPill: propTypes.node.isRequired}`
- `{filterPill: React.ReactNode | null}` to be `{filterPill: propTypes.oneOfType([propTypes.node, propTypes.oneOf([null])])}`

So this is required to suppress the devtools warning.

Reply:
Ah I think this functionality might be the source of the error: https://github.com/milesj/babel-plugin-typescript-to-proptypes#strict-boolean
It might look just a bit nicer if you made it optional instead of having an `| null` if it accepts that. 

Any idea what the prop types conversion gets us? I know there is a runtime check but does it actually catch anything beyond what Typescript would?

Reply:
Ahh, that makes sense. Thanks for the link. 

I don't personally find the auto propTypes very helpful. I'd much rather see a linting or TS error than a runtime error, and it would appear the propTypes conversion isn't even very accurate in cases like this. Given that they're generated from TS, I would think the TS errors would actually be more thorough.

And turning off propTypes generation might speed up development builds.

Reply:
Re: `{filterPil?: React.ReactNode}` vs `{filterPill: React.ReactNode | null}` I don't really care much, but am 1/5 towards the latter, because the state lives in the parent.

I'd propose we leave as-is for now, and re-address when/if we take on any `babel-plugin-typescript-to-proptypes` changes.